src/Files.App/Helpers/RecycleBinHelpers.cs

```diff
@@ -66,12 +61,34 @@ public async Task EmptyRecycleBin()
                 SecondaryButtonText = "Cancel".GetLocalizedResource(),
                 DefaultButton = ContentDialogButton.Primary
             };
-
+            
             ContentDialogResult result = await this.SetContentDialogRoot(ConfirmEmptyBinDialog).ShowAsync();
 
             if (result == ContentDialogResult.Primary)
             {
-                Shell32.SHEmptyRecycleBin(IntPtr.Zero, null, Shell32.SHERB.SHERB_NOCONFIRMATION | Shell32.SHERB.SHERB_NOPROGRESSUI);
+                var banner = App.OngoingTasksViewModel.PostBanner(
+                    "EmptyRecycleBin".GetLocalizedResource(),
+					"EmptyingRecycleBin".GetLocalizedResource(),
+                    0.0f,
+                    ReturnResult.InProgress,
+                    FileOperationType.Delete);
+
+                bool opSucceded = Shell32.SHEmptyRecycleBin(IntPtr.Zero, null, Shell32.SHERB.SHERB_NOCONFIRMATION | Shell32.SHERB.SHERB_NOPROGRESSUI).Succeeded;
+                banner.Remove();
+                if (opSucceded)
+                    App.OngoingTasksViewModel.PostBanner(
+                        "EmptyRecycleBin".GetLocalizedResource(),
+						"BinEmptyingSucceded".GetLocalizedResource(),
+                        100.0f,
+                        ReturnResult.Success,
+                        FileOperationType.Delete);
+                else
+                    App.OngoingTasksViewModel.PostBanner(
+                        "EmptyRecycleBin".GetLocalizedResource(),
+						"BinEmptyingFailed".GetLocalizedResource(),
+                        100.0f,
+                        ReturnResult.Failed,
+                        FileOperationType.Delete);
```

Comment:
To prevent to fetch twice the same localization, let's puts `"EmptyRecycleBin".GetLocalizedResource()` inside a variable: 
```suggestion
                var emptyRecycleBinText = "EmptyRecycleBin".GetLocalizedResource();
                var banner = App.OngoingTasksViewModel.PostBanner(
                    emptyRecycleBinText,
                    "EmptyingRecycleBin".GetLocalizedResource(),
                    0.0f,
                    ReturnResult.InProgress,
                    FileOperationType.Delete);

                bool opSucceded = Shell32.SHEmptyRecycleBin(IntPtr.Zero, null, Shell32.SHERB.SHERB_NOCONFIRMATION | Shell32.SHERB.SHERB_NOPROGRESSUI).Succeeded;
                banner.Remove();
                if (opSucceded)
                    App.OngoingTasksViewModel.PostBanner(
                        emptyRecycleBinText,
                        "BinEmptyingSucceded".GetLocalizedResource(),
                        100.0f,
                        ReturnResult.Success,
                        FileOperationType.Delete);
                else
                    App.OngoingTasksViewModel.PostBanner(
                        emptyRecycleBinText,
                        "BinEmptyingFailed".GetLocalizedResource(),
                        100.0f,
                        ReturnResult.Failed,
                        FileOperationType.Delete);
```

Reply:
That's true, I thought of doing so and then I forgot to write it